utility/namespace.py

```diff
@@ -0,0 +1,29 @@
+""" Create a namespace """
+import os
+from utility import templating
+from ocs.utils import create_oc_resource
+
+
+def create_namespace(name, region='us-east-2'):
+    """
+    Create a namespace
+
+    When a new namespace is created, a directory in /tmp with the namespace's
+    name will be created containing the namespace.yaml file that created this
+    namespace.
+
+    Raises AlreadyExists exception if the namespace already exists.
+
+    Args:
+        name (str): Namespace to be created
+        region (str): AWS region.  Defaults to us-east-2
+    """
+    env_data = dict()
+    env_data['cluster_namespace'] = name
+    env_data['region'] = region
```

Comment:
What does a region do here? I'm wondering how is a k8s namespace related to it?

Reply:
Actually, there is no 'region' information in the generated yaml that is used by the oc command.  I think that I can safely remove it.